spec/requests/lettings_logs_controller_spec.rb

```diff
@@ -1270,33 +1276,33 @@
 
       it "creates an E-mail job" do
         expect {
-          post "/lettings-logs/email-csv", headers:, params: {}
-        }.to enqueue_job(EmailCsvJob).with(user, nil, {}, false)
+          post "/lettings-logs/email-csv?is_codes_only_export=true", headers:, params: {}
+        }.to enqueue_job(EmailCsvJob).with(user, nil, {}, false, nil, true)
       end
 
       it "redirects to the confirmation page" do
-        post "/lettings-logs/email-csv", headers:, params: {}
+        post "/lettings-logs/email-csv?is_codes_only_export=true", headers:, params: {}
         expect(response).to redirect_to(csv_confirmation_lettings_logs_path)
       end
 
       it "passes the search term" do
         expect {
-          post "/lettings-logs/email-csv?search=#{lettings_log.id}", headers:, params: {}
-        }.to enqueue_job(EmailCsvJob).with(user, lettings_log.id.to_s, {}, false)
+          post "/lettings-logs/email-csv?search=#{lettings_log.id}&is_codes_only_export=false", headers:, params: {}
+        }.to enqueue_job(EmailCsvJob).with(user, lettings_log.id.to_s, {}, false, nil, false)
       end
 
       it "passes filter parameters" do
         expect {
-          post "/lettings-logs/email-csv?status[]=completed", headers:, params: {}
-        }.to enqueue_job(EmailCsvJob).with(user, nil, { "status" => %w[completed] }, false)
+          post "/lettings-logs/email-csv?status[]=completed&is_codes_only_export=true", headers:, params: {}
+        }.to enqueue_job(EmailCsvJob).with(user, nil, { "status" => %w[completed] }, false, nil, true)
       end
 
       it "passes a combination of search term and filter parameters" do
         postcode = "XX1 1TG"
 
         expect {
-          post "/lettings-logs/email-csv?status[]=completed&search=#{postcode}", headers:, params: {}
-        }.to enqueue_job(EmailCsvJob).with(user, postcode, { "status" => %w[completed] }, false)
+          post "/lettings-logs/email-csv?status[]=completed&search=#{postcode}&is_codes_only_export=false", headers:, params: {}
+        }.to enqueue_job(EmailCsvJob).with(user, postcode, { "status" => %w[completed] }, false, nil, false)
```

Comment:
- [ ] add test that codes_only parameter is passed in isolation and in conjunction with other parameters